### DISCUSSION TITLE
How do I fetch existing remote posts?

### DISCUSSION THREAD

**Discussion by ghost** [misskey-dev/misskey#7624] [Q&A] [answered]
I just set up a new Misskey instance. Of course, it doesn't have old posts from remote users. How do I fetch the old posts?

**Comment by syuilo** [comment, marked as answer]
GitHub Discussion is not used very often, so please consider using [Misskey Forum](https://forum.misskey.io/) or [Discord](https://discord.gg/QPCwCmZc94).
Thanks!
